torchvision/prototype/models/detection/mask_rcnn.py

```diff
@@ -28,6 +28,9 @@ class MaskRCNN_ResNet50_FPN_Weights(WeightsEnum):
         url="https://download.pytorch.org/models/maskrcnn_resnet50_fpn_coco-bf2d0c1e.pth",
         transforms=CocoEval,
         meta={
+            "task": "image_object_detection",
```

Comment:
I'm unclear on what to put here.
Indeed, this model could also be categorized as `image_instance_segmentation`. Maybe having `task` be a list would be of help here?

Ensuring that this is adapted with paperswithcode categorization would be good as well https://paperswithcode.com/paper/mask-r-cnn

Reply:
Yeah same here. I'll leave as is for now and review in more detail on the future.